src/blockchain/node/UpdateTransaction.cpp

```diff
@@ -123,11 +116,7 @@ auto UpdateTransaction::disconnected() const noexcept
 auto UpdateTransaction::EffectiveBestBlock(const block::Height height) const
     noexcept(false) -> block::Hash
 {
-    try {
-
-        return best_.at(height);
-    } catch (...) {
-    }
+    if (best_.count(height)) return best_.at(height);
```

Comment:
Don't know if it is possible with that robin hood map, but generally, for the sake of efficiency, one would prefer
if(auto i = best_.find(height); i != best_.end()) return i->second;

Reply:
definitelly. I'll change it in other places where I used this pattern too